<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SciFilter</class>
 <widget class="QWidget" name="SciFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SciFilter</string>
  </property>
  <layout class="QHBoxLayout" name="hLayout">
   <item>
    <layout class="QVBoxLayout" name="vLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QGroupBox" name="grpArch">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>结构</string>
       </property>
       <layout class="QHBoxLayout" name="hLayoutArch">
        <item>
         <widget class="QRadioButton" name="chkFIR">
          <property name="text">
           <string>FIR</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">bgrpArch</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="chkIIR">
          <property name="text">
           <string>IIR</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">bgrpArch</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="grpType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>类型</string>
       </property>
       <layout class="QGridLayout" name="gLayoutType">
        <item row="0" column="0">
         <widget class="QRadioButton" name="chkLP">
          <property name="text">
           <string>Low Pass</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">bgrpType</string>
          </attribute>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QRadioButton" name="chkBP">
          <property name="text">
           <string>Band Pass</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">bgrpType</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="chkHP">
          <property name="text">
           <string>High Pass</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">bgrpType</string>
          </attribute>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QRadioButton" name="chkBS">
          <property name="text">
           <string>Band Stop</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">bgrpType</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="grpModl">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>模型</string>
       </property>
       <layout class="QGridLayout" name="gLayoutModl">
        <item row="1" column="0">
         <widget class="QRadioButton" name="chkButt">
          <property name="text">
           <string>Butterworth</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">bgrpModl</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="chkCheb">
          <property name="text">
           <string>Chebyshev</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">bgrpModl</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="chkBess">
          <property name="text">
           <string>Bessel</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">bgrpModl</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QRadioButton" name="chkElli">
          <property name="text">
           <string>Elliptic</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">bgrpModl</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hLayout1">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="lblFsample">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>采样频率：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="linFsample">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>2500Hz</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hLayout2">
       <item>
        <widget class="QLabel" name="lblFstop">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>截止频率：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="linFstop">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>200Hz</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hLayout3">
       <item>
        <widget class="QLabel" name="lblHlimit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>上限频率：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="linHlimit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>500Hz</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hLayout4">
       <item>
        <widget class="QLabel" name="lblNtap">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>抽头  数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="linNtap">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>60</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hLayout5">
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnCalc">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>计算绘图</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="vSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGraphicsView" name="gView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="bgrpArch"/>
  <buttongroup name="bgrpType"/>
  <buttongroup name="bgrpModl"/>
 </buttongroups>
</ui>
